packager/react-native-xcode.sh

```diff
@@ -93,7 +93,7 @@ $NODE_BINARY "$REACT_NATIVE_DIR/local-cli/cli.js" bundle \
   --bundle-output "$BUNDLE_FILE" \
   --assets-dest "$DEST"
 
-if [[ ! $DEV && ! -f "$BUNDLE_FILE" ]]; then
```

Comment:
Hmm...the prior check would handle the case where `$DEV` is undefined (eg `react-native-xcode.sh`) whereas the new check would handle `$DEV` being explicitly defined as false (eg `$DEV=false react-native-xcode.sh`). I wonder if we shouldn't actually be checking for `$DEV != true`? since that would handle both cases.